src/main/telemetry/ghst.c

```diff
@@ -112,10 +113,49 @@ void ghstFramePackTelemetry(sbuf_t *dst)
     sbufWriteU8(dst, 0x00);                     // tbd3
 }
 
+// GPS data, primary, positional data
+void ghstFrameGpsPrimaryTelemetry(sbuf_t *dst)
+{
+    // use sbufWrite since CRC does not include frame length
+    sbufWriteU8(dst, GHST_FRAME_GPS_PAYLOAD_SIZE + GHST_FRAME_LENGTH_CRC + GHST_FRAME_LENGTH_TYPE);
+    sbufWriteU8(dst, GHST_DL_GPS_PRIMARY);
+
+    sbufWriteU32(dst, gpsSol.llh.lat);
+    sbufWriteU32(dst, gpsSol.llh.lon);
+
+    // constrain alt. from -32,000m to +32,000m, units of meters
+    const int16_t altitude = (constrain(getEstimatedAltitudeCm(), -32000 * 100, 32000 * 100) / 100);
```

Comment:
Note that the output of `getEstimatedAltitudeCm()` will vary between absolute altitue when unarmed and altitude relative to arming point when armed - this is potentially not idea for use in a GPS data frame, and we might want to consider to sending the GPS based altitude, or offsetting the altitude by the absolute altitude at the arming point.
Conversely, if an 'non-GPS-altitude' frame is added, this should probably return relative altitude.

Reply:
Mike, while I agree that the current implementation is not perfect, it appears to be commonly used by other protocols, and therefore expected by users. 
Equivalent drivers have been already merged into other open/closed source coded (including iNav), and to have different behavior in BF at this point would add more confusion than it removes. 

Depending upon the use-case, pilots may be interested in either altitude relative to arming point, or absolute altitude. Providing both with a future update to the OTA protocol, and the various FC drivers probably makes the most sense, not only for GHST, but for any of the BF Rx protocol drivers.

Reply:
@tonycake: Telemetry protocols that support GPS information, like FrSky SmartPort, return absolute altitude as part of the GPS data frame, both in Betaflight and in iNav.

Reply:
Mike, is this going to block this merge? To add what you are requesting I need to add a new OTA packet, release new firmware for Ghost, and add support it in OpenTx, iNav (there is no space left in the primary and secondary GPS packets). 
Since there is a round of changes due after 4.3 is released to add some new features, I would rather slide this absolute altitude feature into that change.